xml/ha_config_hawk2.xml

```diff
@@ -269,28 +269,31 @@
       <itemizedlist>
        <listitem>
         <para>
-         <guimenu>Add resource</guimenu>: Opens the resource configuration
+         <guimenu>Add Resource</guimenu>: Opens the resource configuration
          screen. For details, see <xref linkend="sec.conf.hawk2.rsc"/>.
         </para>
        </listitem>
        <listitem>
         <para>
-         <guimenu>Add constraint</guimenu>: Opens the constraint configuration
+         <guimenu>Add Constraint</guimenu>: Opens the constraint configuration
          screen. For details, see <xref linkend="sec.conf.hawk2.cons"/>.
         </para>
        </listitem>
        <listitem>
         <para>
-         <guimenu>Wizard</guimenu>: Allows you to select from several wizards
+         <guimenu>Wizards</guimenu>: Allows you to select from several wizards
          that guide you through the creation of resources for a certain
          workload, for example, a DRBD block device. For details, see
          <xref linkend="sec.conf.hawk2.rsc.wizard"/>.
         </para>
        </listitem>
        <listitem>
         <para>
-         <guimenu>Edit Configuration</guimenu>: Allows you to edit node names
-         and attributes, resources, constraints, and tags.
+         <guimenu>Edit Configuration</guimenu>: Allows you to edit resources,
+         constraints, node names and attributes, tags, alerts, and fencing
+         topologies.
+         <remark>taroth 2017-08-10: what are the new entries, 'Alerts'
+          and 'Fencing' good for?</remark>
```

Comment:
Thanks for the information! As Fate only requested a RN entry for both features, I only added this to the remark for now and added a note to myself to probably say a bit more about this for next release.